chapters/git-cmdline.Rmd

```diff
@@ -876,7 +876,7 @@ If we view our GitHub repository in the browser,
 it now includes all of our project files,
 along with all of the commits we have made so far (Figure \@ref(fig:git-cmdline-history)).
 
-```{r git-cmdline-history, echo=FALSE, fig.cap="Repository history on GitHub"}
+```{r git-cmdline-history, echo=FALSE, fig.cap="Viewing Repository history on GitHub"}
```

Comment:
```suggestion
```{r git-cmdline-history, echo=FALSE, fig.cap="Viewing the repository's history on GitHub"}
```